src/main/convert.js

```diff
@@ -60,30 +58,33 @@ export class ThriftFileConverter {
       [
         '// @flow',
         `// Generated by thrift2flow at ${new Date().toString()}${
-          this.withsource ? `\n// Source: ${this.thriftPath}` : ''}`,
+          this.withsource ? `\n// Source: ${this.thriftPath}` : ''
+        }`,
         this.generateImports(),
-        ...this.thrift.asts[this.thrift.filename].definitions.map(this.convertDefinitionToCode)
+        ...this.thrift.asts[this.thrift.filename].definitions.map((value, index, array) =>
```

Comment:
I think this code would look a lot nicer (and this diff would be a lot smaller :) ) if you just stored this as a field, or even just declared a getter function for this value.